python/lsst/ip/diffim/subtractImages.py

```diff
@@ -167,7 +172,59 @@ def __init__(self, **kwargs):
         self.convolutionControl.setDoNormalize(False)
         self.convolutionControl.setDoCopyEdge(True)
 
-    def run(self, template, science, sources):
+    @lsst.utils.inheritDoc(lsst.pipe.base.PipelineTask)
+    def runQuantum(self, butlerQC: lsst.pipe.base.ButlerQuantumContext,
+                   inputRefs: lsst.pipe.base.InputQuantizedConnection,
+                   outputRefs: lsst.pipe.base.OutputQuantizedConnection):
+        inputs = butlerQC.get(inputRefs)
+        self.log.info("Processing %s", butlerQC.quantum.dataId)
+
+        if self.config.doApplyFinalizedPsf:
+            finalizedPsfApCorrCatalog = inputs.get("finalizedPsfApCorrCatalog", None)
+            inputs["science"] = self.prepareCalibratedExposure(
+                inputs["science"],
+                finalizedPsfApCorrCatalog=finalizedPsfApCorrCatalog
+            )
+        subtractResults = self.run(**inputs)
+        return subtractResults
+
+    def prepareCalibratedExposure(self, exposure, finalizedPsfApCorrCatalog=None):
```

Comment:
It's not a calibrated exposure; maybe `_applyExternalCalibrations` (since this seems like a private-ish method)?